api/staged_products_property_collection_guid_assigner.go

```diff
@@ -137,12 +156,20 @@ func (existingCollection responsePropertyCollection) findGUIDForEquivalentlItem(
 	return "", false
 }
 
-func associateExistingCollectionGUIDs(updatedProperty interface{}, existingProperty ResponseProperty) error {
-	updatedCollection, err := parseUpdatedPropertyCollection(updatedProperty)
+type associateExistingCollectionGUIDsInput struct {
```

Comment:
Most of the changes are a result of this refactoring, implemented to make passing the properties needed to enable `func parseResponsePropertyCollection()` to make a `Api.GetDeployedProductCredential()` call to fetch the existing secret values